tutorials/RecSys22tutorial/03_Customize_Merlin_Models.ipynb

```diff
@@ -36,7 +36,7 @@
    "id": "ebc9c8c6-9fab-435f-958b-923b0f3d6ff4",
```

Comment:
<p>Can we be a little bit more detailed in that example?</p><ul><li>print the batch to see, what the output is</li><li>print continuous_block(batch) to see the output</li><li>print ottom_block(batch) to see the output</li></ul><p> </p><p>Can we add more explicit a text that we can feed an example batch through Merlin Models layers and get the results? I think I missed that in https://github.com/NVIDIA-Merlin/models/blob/main/examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb and that is super important to debug/build your own architecture</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/publications/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>ok.</p>